packages/ui/src/components/timeline/Timeline.module.scss

```diff
@@ -283,6 +275,15 @@
   }
 }
 
+.rangeContainer {
+  height: 32px;
+  pointer-events: none;
```

Comment:
The track itself should have a `pointer` cursor to signify that it can be edited:
```suggestion
  pointer-events: none;
  cursor: pointer;
```

Reply:
I agree, but I think the cursor should be changed only when the `rangeTrack` is active (i.e. when `SHIFT` and `CTRL` are pressed)

Reply:
Exactly, the cursor won't be changed when `pointer-events` is set to `none`

Reply:
Oh yeah of course youre right, didn't think of that. My bad :)